fastlane-plugin-revenuecat_internal.gemspec

```diff
@@ -24,7 +24,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency('rest-client')
 
   spec.add_development_dependency('bundler')
-  spec.add_development_dependency('fastlane', '2.207.0')
+  spec.add_development_dependency('fastlane')
```

Comment:
Needed latest _fastlane_ due to fixes for Xcode 16.3

Reply:
Even if we need to update `fastlane`, shouldn't we keep it pinned to a specific version?

Reply:
> Even if we need to update `fastlane`, shouldn't we keep it pinned to a specific version?

IMO, yes and no 🤷‍♂️ I caused issues for me being pinned to a specific version here. I can change this to be `>=` since there are some fixes needed in new version for `xcresulttool` so I will probably do that